app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/ui/notification/DeviceShieldReminderNotificationScheduler.kt

```diff
@@ -67,6 +67,7 @@ class DeviceShieldReminderNotificationScheduler @Inject constructor(
         vpnStopReason: VpnStopReason
     ) {
         when (vpnStopReason) {
+            VpnStopReason.RESTART -> {} // no-op
```

Comment:
I don't think we need to do anything here when we do a restart, let me know if it's not the case.